src/main/java/io/zold/api/Score.java

```diff
@@ -24,18 +24,81 @@
 
 package io.zold.api;
 
+import com.github.victornoel.eo.GenerateEnvelope;
 import org.cactoos.Text;
+import org.cactoos.iterable.Joined;
+import org.cactoos.iterable.LengthOf;
+import org.cactoos.iterable.Mapped;
 
 /**
  * A remote node's score, equal to its number of suffixes.
  *
+ * The natural order of {@link Score} is from highest to lowest.
+ *
+ * Note: {@link Score} has a natural ordering that is inconsistent with equals.
+ *
  * @since 0.1
  */
+@GenerateEnvelope
 public interface Score extends Comparable<Score> {
+
     /**
      * The suffixes associated with this score. Each suffix is a text of the
      * form {@code /[a-zA-Z0-9]+/}.
      * @return Suffixes for this score
      */
     Iterable<Text> suffixes();
+
+    /**
+     * Default implementation for {@link Score}.
+     *
+     * @since 1.0
+     */
+    final class Default implements Score {
+
+        /**
+         * The suffixes.
+         */
+        private final Iterable<Text> suffixes;
+
+        /**
+         * Ctor.
+         *
+         * @param suffixes The suffixes.
+         */
+        Default(final Iterable<Text> suffixes) {
+            this.suffixes = suffixes;
+        }
+
+        @Override
+        public int compareTo(final Score other) {
+            return new LengthOf(other.suffixes()).intValue()
+                - new LengthOf(this.suffixes).intValue();
+        }
+
+        @Override
+        public Iterable<Text> suffixes() {
+            return this.suffixes;
+        }
+    }
+
+    /**
+     * Summary of multiple {@link Score}.
+     *
+     * @since 1.0
+     */
+    final class Sum extends ScoreEnvelope {
```

Comment:
@victornoel I think this can be substituted for cactoo's `SumOf` where it's currently being used

Reply:
@llorllale The place where it is used (in `Copy.score()`) currently returns a `Score`, not a number.

Do you want me to change the return type to `int` or did you have something else in mind?

Reply:
@victornoel alright, let's keep `Sum` but rename it to `Summed`.

Reply:
@llorllale ok